src/js/providers/html5.js

```diff
@@ -229,13 +244,20 @@ function VideoProvider(_playerId, _playerConfig) {
         }
     }
 
+    function _setPositionBeforeSeek(position) {
+        _positionBeforeSeek = _convertTime(position);
+    }
+
     _this.getCurrentTime = function() {
-        let currentTime = _videotag.currentTime;
+        return _convertTime(_videotag.currentTime);
+    };
+
+    function _convertTime(position) {
         if (_this.getDuration() < 0) {
-            currentTime = -(_getSeekableEnd() - currentTime);
+            position = -(_getSeekableEnd() - position);
```

Comment:
this is essentially 
`position = position -_getSeekableEnd();` or 
`position -= _getSeekableEnd()`;